corehq/apps/reports/standard/deployments.py

```diff
@@ -319,3 +323,82 @@ def _bootstrap_class(obj, severe, warn):
         return "label label-warning"
     else:
         return "label label-success"
+
+
+class ApplicationErrorReport(GenericTabularReport, ProjectReport):
+    name = ugettext_noop("Application Error Report")
```

Comment:
These are forcibly translated later ([here](https://github.com/dimagi/commcare-hq/blob/31c03b14ed7bf79fa240e29f147e376829bfcd1b/corehq/apps/reports/generic.py#L299-L300), I think).  The other report classes also use `ugettext_noop`